src/kv/generic_serialise_wrapper.h

```diff
@@ -50,6 +50,7 @@ namespace kv
     W private_writer;
     W* current_writer;
     Version version;
+    Version max_conflict_version;
```

Comment:
As it is not updated automatically yet, could you update the table in https://microsoft.github.io/CCF/master/build_apps/kv/kv_serialisation.html#serialised-transaction-format as well?